pkgs/development/python-modules/diffimg/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pillow
+, unittestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "diffimg";
+  version = "0.3.0"; # github recognized 0.1.3, there's a v0.1.5 tag and setup.py says 0.3.0
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "nicolashahn";
+    repo = "diffimg";
+    rev = "b82f0bb416f100f9105ccccf1995872b29302461";
+    hash = "sha256-H/UQsqyfdnlESBe7yRu6nK/0dakQkAfeFaZNwjCMvdM=";
+  };
+
+  postPatch = ''
+    substituteInPlace diffimg/test.py \
+      --replace "from diff import diff" "from diffimg.diff import diff"
+  '';
```

Comment:
A comment here briefly explaining why this is needed would be helpful for anyone who comes along later.

Reply:
good point, done